resources/keywords/provision_dbinstance.resource

```diff
@@ -29,6 +29,18 @@ Toggle Database Provider Drop Down ${isv_name}
     Select From List By Label    ${drpDwnDBProvider}    ${isv_name}
     Element Should Be Visible    ${drpDwnProvAccOptn}
 
+User Selects DBProvider And Provider Account with RDS DB Engine
```

Comment:
This can be achieved with the existing keyword "User Selects DBProvider And Provider Account". In "Toggle Database Provider Drop Down" add an if condition to match "RDS" in the variable "${isv_name}". If the question is true, then perform the action "Wait Until Keyword Succeeds 10s 1s Toggle RDS Database ${engine} Drop Down"